File: app/src/main/java/com/example/openfoodfactesgi/dto/DTOSuccess.java
package com.example.openfoodfactesgi.dto;
import com.google.gson.annotations.SerializedName

data class DTOSuccess (
        @SerializedName("name") var name: String,
        @SerializedName("isClient") var isClient: Boolean,
        @SerializedName("isBu") var isBu: Boolean,
        @SerializedName("isBuOrigine") var isBuOrigine: Boolean,
        @SerializedName("idPlanning") var idPlanning: Int,
        @SerializedName("nom") var nom: String?
){
}

File: app/src/main/java/com/example/openfoodfactesgi/dto/DTOSuccess.kt
package com.example.openfoodfactesgi.dto

import com.google.gson.annotations.SerializedName

data class DTOSuccess(
    @SerializedName("idSuccess") var idSuccess: String,
    @SerializedName("name") var name: String,
    @SerializedName("description") var description: String,
    @SerializedName("image") var image: String,
) {
}
